app/assets/src/components/views/discovery/mapping/map_preview_sidebar.scss

```diff
@@ -48,7 +48,24 @@
 
 .tableDataRow {
   align-items: flex-start;
-  padding: 11px 0;
+  padding: 10px 0;
+  margin-left: 0;
+
+  // Override defaults
+  &:global(.ReactVirtualized),
```

Comment:
There are props to pass class names to customize this I believe. Can you search within `BaseTable` and `ReactVirtualize` libraries?

Reply:
yes it looks like we get headerClassName and rowClassName https://github.com/bvaughn/react-virtualized/blob/master/docs/customizingStyles.md

I moved out the header but wasn't able to override rowColumn/headerColumn specifically

![Screen Shot 2019-06-19 at 2 46 48 PM](https://user-images.githubusercontent.com/5652739/59804083-36027a00-92a2-11e9-954b-61ceeb9217be.png)